packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -176,3 +160,18 @@ function isGroup(option: AnyOption) {
   if (option.options) return true;
   return false;
 }
+
+function useRepositionMenu(attachTo: MenuProps["attachTo"]) {
+  const [positionElement, setPositionedElementRef] =
+    useState<HTMLElement | null>();
```

Comment:
Shouldn't this be `useRef` instead?

Reply:
probably. I copied this from another place where it actually needs to use a state since it gets unmounted and mounted back in when it shows up. But this whole component gets unmounted anyway. so probably.

Reply:
`useRef` breaks it as we saw